### ISSUE TITLE
Cancelling a delete form should take user back to where they came from vs ListView

### ISSUE THREAD

**Issue by jmichelsen** [jmichelsen/bisl__backend#59]

**Comment by kupchoy** [comment]
I believe this is already been implemented

**Comment by jmichelsen** [comment]
in master, this currently still sends users to `ListView`
```
                    <a class="btn btn-outline-primary btn-sm" href="{% url "recipes:list" %}">Cancel</a>

```

**Comment by kupchoy** [comment]
Yes and that's what I was suggesting to do since the delete button there.  Previously hitting cancel sent you to detail view.  Jonathan changed it in this PR
[https://github.com/jmichelsen/bisl__backend/pull/58/files/fd0352fc5c7c4b37e9158875934de764a268399d..6ba9b7205328a681b47f0eef20557c9669b487dd#diff-71b6623c81d9ba53a9955abe27f7f551](url)

**Comment by jmichelsen** [comment]
well, you get to the deleteview from the detailview, so cancel should send you back to the detailview

**Comment by kupchoy** [comment]
I am pretty sure the delete view originates from the list view.  I think the confusion is the title of this issue is backwards accidentally.  This is the table that I had posted on the review of the PR

![image](https://user-images.githubusercontent.com/48184608/67953378-60a3fe00-fbc5-11e9-9bf4-c216d1fcc8d5.png)

**Comment by jmast02** [comment]
Alright guys, I changed it so that the `delete` button is only on the DetailView, so then when you do hit delete, it takes you to the confirm delete page. From there, when you hit cancel, it takes us back to the DetailView, where we previously were. Should make more sense now.
